WordPress/WordPressNotificationContentExtension/Sources/NotificationViewController.swift

```diff
@@ -2,11 +2,66 @@ import UIKit
 import UserNotifications
 import UserNotificationsUI
 
-class NotificationViewController: UIViewController, UNNotificationContentExtension {
+// MARK: - NotificationViewController
 
-    @IBOutlet var label: UILabel?
+/// Responsible for enhancing the visual appearance of designated push notifications.
+@objc(NotificationViewController)
```

Comment:
Does this need to be @objc? Or at the least, I don't think we need the class name afterwards.

Reply:
This is marked Objective-C so that it's more easily found when the extension is bootstrapped. I believe the alternative we have is to prefix the type name with the Swift module name in `Info.plist`.

I thought this was more approachable. I'd welcome your thoughts.